src/styles/block/card.scss

```
@@ -0,0 +1,106 @@
+.card {
+  width: $card-width;
+  padding: 32px $card-padding $card-padding;
+
+  background-color: $white-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  transition: transform $transition-duration $transition-type;
+    color: $transition-duration $transition-type;
+
+  &:hover {
+    transform: scale(1.2);
+  }
```

color doesn't change with :hover
```suggestion
  transition: transform $transition-duration $transition-type; 

  &:hover {
    transform: scale(1.2);
  }
```

---

done